tests/test_settings.py

```diff
@@ -165,6 +165,24 @@ def test_changelog():
     assert s.build_tag("bar", "2.1.0") == ()
 
 
+def test_global_changelog():
+    s = _parse(
+        textwrap.dedent("""
+    changelog:
```

Comment:
We'll want the changelog to have a variant component so that a change to one variant doesn't trigger a complete rebuild of all of the other variants.

We should probably make the per-package changelog variant-specific, too.

Reply:
Should we address this once #323 lands cause it defines a per-package variant specific settings and it might be easier to build on top of that?

Reply:
Yeah, we can put off the per-package change until after 323, but let's make the global list variant-specific here.